sentry-android-core/src/main/AndroidManifest.xml

```diff
@@ -2,7 +2,8 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android">
   <uses-permission android:name="android.permission.INTERNET"/>
 
-  <application>
+  <application
+    android:supportsRtl="true">
```

Comment:
do we need to set `supportsRtl`? I'm wondering if this could cause manifest merging issues.